test/Masa.Contrib.Dispatcher.Events.Tests/FeaturesTest.cs

```diff
@@ -22,7 +22,7 @@ public async Task TestMethodsReturnType()
             {
                 ResetMemoryEventBus(typeof(AddBasketEvent).Assembly);
             }
-            catch (Exception)
+            catch (Exception ex)
```

Comment:
The variable 'ex' is declared but never used

Reply:
Processed